install/install-kritis.sh

```diff
@@ -38,17 +38,14 @@ while getopts "n:s" opt; do
   esac
 done
 
-# creates a cert
-function kritis::create_cert {
-    # First Substitute $NAMESPACE in kritis-charts/cert.yaml
-    cat $CERT_TEMPLATE_FILE | sed "s/NAMESPACE/$NAMESPACE/g" > $CERT_FILE
-    CMD="helm certgen generate $CHARTS_DIR --namespace $NAMESPACE"
-    $CMD
+function kritis::preinstall {
+  cat $PREINSTALL_TEMPLATE_FILE | sed "s/NAMESPACE/$NAMESPACE/g" > $PREINSTALL_FILE
```

Comment:
You can drop this templating by  leaving namespace out all together and passing it to the kubectl apply -f below.